src/main/java/net/pms/configuration/RendererConfiguration.java

```diff
@@ -2119,7 +2119,7 @@ public String toString() {
 	 * @return whether to use MediaInfo
 	 */
 	public boolean isUseMediaInfo() {
-		return getBoolean(MEDIAPARSERV2, false) && LibMediaInfoParser.isValid();
+		return getBoolean(MEDIAPARSERV2, true) && LibMediaInfoParser.isValid();
```

Comment:
After reading the chained constructors I found what I believe to be the default setting, let me know if I misunderstood or made a bad change @SubJunk @valib